ui/src/progress_table_cell_type.tsx

```diff
@@ -17,19 +17,42 @@ import { F, S } from './core'
 import React from 'react'
 import { stylesheet } from 'typestyle'
 import { ProgressArc } from './parts/progress_arc'
+import { ProgressBar } from './parts/progress_bar'
 import { clas, cssVar } from './theme'
 
 const
   css = stylesheet({
-    container: {
+    barContainer: {
       position: 'relative',
+      display: 'inline-flex',
+      width: 100,
```

Comment:
Not sure what you mean about having the width be fluid.
The existing code before I changed anything had the width and height set to 50, which is the size of the spinner (this is what increased the row height of the table). I used this idea to create the bar, setting the bar width to 50 as well. 
I'll remove the barContainer's width and use a margin for making the percentage sign appear next to the bar instead of setting it as 100. Is that ok?